src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS/mc_sseg_stime_NOLOOP_demo.py

```diff
@@ -1,4 +1,6 @@
 import traceback
+from troute.routing.fast_reach import reach
+from test_suite_parameters import generate_conus_MC_parameters
 
 debuglevel = 0
 COMPILE = True
```

Comment:
@groutr: Could this easily be turned into an input?

Reply:
I don't see that being easy if not impossible. It would have to exist at a "meta-test" level.